frontend/app/services/work-package-service.js

```diff
@@ -221,9 +261,11 @@ module.exports = function($http,
     },
 
     updateWorkPackage: function(workPackage, notify) {
+      console.log(workPackage.links.updateImmediately);
```

Comment:
Do we usually keep such statements? Looks like printf debugging, but maybe I just don't see the reoccuring value in this log output.